nautobot/extras/models/jobs.py

```diff
@@ -538,6 +538,13 @@ class Meta:
         get_latest_by = "created"
         verbose_name_plural = "job log entries"
 
+        indexes = [
+            models.Index(
+                name="extras_joblogentry_created_idx",
+                fields=["created"],
+            ),
+        ]
```

Comment:
Could we just add `db_index=True` to the `created` field declaration above, instead?